bundles/AdminBundle/Resources/public/js/pimcore/settings/thumbnail/item.js

```diff
@@ -1348,6 +1348,35 @@ pimcore.settings.thumbnail.items = {
             }]
         });
 
+        return item;
+    },
+
+    item1x1Placeholder: function (panel, data, getName) {
+
+        var niceName = t("1x1_pixel_placeholder");
+        if (typeof getName != "undefined" && getName) {
+            return niceName;
+        }
+
+        if (typeof data == "undefined") {
+            data = {};
+        }
+        var myId = Ext.id();
+
+        var item = new Ext.form.FormPanel({
+            id: myId,
+            style: "margin-top: 10px",
+            border: true,
+            bodyStyle: "padding: 10px;",
+            tbar: this.getTopBar(niceName, myId, panel),
+            html: t("1x1_pixel_placeholder_description"),
+            items: [{
+                xtype: "hidden",
+                name: "type",
+                value: "1x1_pixel"
```

Comment:
value should be "1x1Placeholder" to match with function name to load save config. https://github.com/pimcore/pimcore/blob/bfcce7ca94c08f30fbca69831a13be574c7613d9/bundles/AdminBundle/Resources/public/js/pimcore/settings/thumbnail/item.js#L263

Reply:
Good catch, thanks a lot!

Reply:
With 8b00edbccb31c90027a42bc7e7354afc6bfcc150 it should be fine now.